### PR TITLE
Fix issues july2015

### PR DESCRIPTION
**Pull request by jmarcelogimenez**
It solves issues:

#35, #36, #37, #41

### CODE REVIEW

**Comment by jmarcelogimenez** [comment]
Maybe I've managed wrongly this PR.  I just tried to create the PR and, because the branch was some steps behind the master, I had to merge it manually by command line. Following the sequence of steps suggested by github the branch was directly merged, without any revision of the rest of the developers....

**Comment by nathanfranklin** [comment]
> Maybe I've managed wrongly this PR. I just tried to create the PR and, because the branch was some steps behind the master, I had to merge it manually by command line. Following the sequence of steps suggested by github the branch was directly merged, without any revision of the rest of the developers.... 

yeah, it looks like this branch was merged without review by any other developers.  

maybe @jmarcelogimenez  and @khiltunen can look if you want to either (i) revert these changes from master and then create a new PR or (ii) leave these change in master and then create a PR to fix any comments

File: foam_internalwrapper/foam_mesh.py [hunk 1]
```diff
@@ -385,126 +398,16 @@ def add_cell(self, cell):
...
-                dataName = "U"
-                foamface.setPointVectorData(self.name,
-                                            self._uuidToFoamLabel[point.uid],
-                                            dataName,
-                                            point.data[data])
-            else:
-                error_str = "Data named "+data+" not supported"
-                raise NotImplementedError(error_str)
+        message = 'Point update not supported yet'
+        raise NotImplementedError(message)
```
Comment (nathanfranklin) [inline foam_internalwrapper/foam_mesh.py]: is `update_point` no longer supported?

File: foam_internalwrapper/foam_mesh.py [hunk 2]
```diff
@@ -385,126 +398,16 @@ def add_cell(self, cell):
...
-            elif data == CUBA.VELOCITY:
-                dataName = "U"
-                foamface.setFaceVectorData(self.name,
-                                           self._uuidToFoamLabel[face.uid],
-                                           dataName,
-                                           face.data[data])
-            else:
-                error_str = "Data named "+data+" not supported"
-                raise NotImplementedError(error_str)
+        message = 'Face update not supported yet'
```
Comment (nathanfranklin) [inline foam_internalwrapper/foam_mesh.py]: is `update_face` no longer supported?

File: foam_controlwrapper/examples/poiseuille.py [hunk 3]
```diff
@@ -41,5 +41,6 @@
 print "Case directory ", mesh_inside_wrapper.path
 
 # run returns the latest time
```
Comment (nathanfranklin) [inline foam_controlwrapper/examples/poiseuille.py]: these 3 commented out lines should be removed

File: foam_controlwrapper/examples/poiseuille_vof.py [hunk 4]
```diff
@@ -67,5 +67,6 @@
     mesh_inside_wrapper.update_cell(cell)
 
 # run returns the latest time
-lastTime = wrapper.run()
-print "Simulation run up to time: ", lastTime
+# lastTime = wrapper.run()
```
Comment (nathanfranklin) [inline foam_controlwrapper/examples/poiseuille_vof.py]: these 3 commented out lines should be removed

File: foam_controlwrapper/foam_controlwrapper.py [hunk 5]
```diff
@@ -103,7 +103,7 @@ def run(self):
 
         # save timestep to mesh
         mesh._time = runner.get_last_time()
-        return mesh._time
+#       return mesh._time
```
Comment (nathanfranklin) [inline foam_controlwrapper/foam_controlwrapper.py]: remove

File: foam_internalwrapper/examples/poiseuille.py [hunk 6]
```diff
@@ -52,8 +52,7 @@
     mesh_inside_wrapper.update_cell(cell)
 
 # run returns the latest time
```
Comment (nathanfranklin) [inline foam_internalwrapper/examples/poiseuille.py]: this outdated comment should be removed

File: foam_internalwrapper/foam_internalwrapper.py [hunk 7]
```diff
@@ -73,7 +73,7 @@ def run(self):
 
         mesh._time = foamface.run(mesh.name, ncores)
 
-        return mesh._time
+#       return mesh._time
```
Comment (nathanfranklin) [inline foam_internalwrapper/foam_internalwrapper.py]: remove commented out line

**Pull request merged**